metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -197,9 +174,11 @@ final class RenameProvider(
       loc <- definitionProvider
         .fromSymbol(companionSymbol)
         .asScala
-      if loc.getUri().isScalaFilename
+      if loc.getUri.isScalaFilename
       companionLocs <- referenceProvider
-        .references(toReferenceParams(loc, includeDeclaration = false))
+        .references(
+          referenceProvider.toReferenceParams(loc, includeDeclaration = false)
```

Comment:
`toReferenceParams` will not be needed here if we switch to maybe using `locations` in `references` method ?